webapp/src/components/Modal/AuctionModal/AuctionModal.container.js

```diff
@@ -0,0 +1,21 @@
+import { connect } from 'react-redux'
+
+import { locations } from 'locations'
+import { navigateTo } from '@dapps/modules/location/actions'
+import { getAddress } from 'modules/wallet/selectors'
+import { getData as getAuthorizations } from 'modules/authorization/selectors'
+
+import AuctionModal from './AuctionModal'
+
+const mapState = state => {
+  const address = getAddress(state)
+  const authorization = getAuthorizations(state)[address]
+
```

Comment:
If we don't have a case where we care about authoriazations of not the current address. What do you think of moving `getAddress(state)` as part of the` getAuthorizations` selector?

We could avoid the need of doing:

```javascript
const address = getAddress(state)
const authorization = getAuthorizations(state)[address]
```

And do just 
```javascript
const authorization = getAuthorizations(state)
```

Reply:
Hmm, we may need authorisations for another address, but I guess it's not that common. I'm not sure about coupling a method name like `getAuthorizations` with `getAddress` but I see your point.

I cast @cazala 🧙‍♂️. What do you think?

Reply:
Ok! I checked and we're already doing this in a couple of places so I'll go with that!